functions.php

```diff
@@ -25,6 +25,11 @@ function elementor_hello_theme_setup() {
 // Theme Scripts & Styles
 function elementor_hello_theme_scripts_styles() {
 	wp_enqueue_style( 'elementor-hello-theme-style', get_stylesheet_uri() );
+
+	// Comment Reply Script
+	if ( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
```

Comment:
It should be in the comments.php file:
https://github.com/pojome/elementor-hello-theme/blob/master/comments.php